Firebase/Messaging/FIRMessagingDefines.h

```diff
@@ -45,3 +45,14 @@ _Pragma("clang diagnostic pop")
     return nil; \
   } while (0)
 #endif
+
+#if defined(DEBUG)
+#define FIRMesssaging_MUST_NOT_BE_MAIN_THREAD()                                                 \
+  do {                                                                                \
+    NSAssert(![NSThread isMainThread], @"Must not be executing on the main thread."); \
```

Comment:
Though it's debug-only, this assert may trigger false-positives if the main GCD queue is suspended. See http://blog.benjamin-encz.de/post/main-queue-vs-main-thread/

Reply:
Removed it for now.